dev/conductor/core/lib/src/globals.dart

```diff
@@ -8,6 +8,8 @@ import 'proto/conductor_state.pb.dart' as pb;
 
 const String gsutilBinary = 'gsutil.py';
 
+const String kFrameworkDefaultBranch = 'master';
```

Comment:
How will this be handled when engine moves to main?

Reply:
This only relevant on the framework, as we only apply tags to the framework.